cocos/network/SocketIO.cpp

```diff
@@ -116,10 +430,10 @@ SIOClientImpl::~SIOClientImpl()
 
 void SIOClientImpl::handshake()
 {
-    log("SIOClientImpl::handshake() called");
+    CCLOGINFO("SIOClientImpl::handshake() called");
 
     std::stringstream pre;
-    pre << "http://" << _uri << "/socket.io/1";
+    pre << "http://" << _uri << "/socket.io/1/?EIO=2&transport=polling&b64=true";
```

Comment:
Hi, @himynameschris 

Our team report a bug on Android shows that this URL is not reachable in the server. Indeed `/socket.io/1/` is not a regular path on any server. I couldn't understand why handshake need such a strange path, so I looked back and found the initial implementation was contributed by you.

I want to ask, why this specific path for handshake ?